client/src/components/layout/mobile.tsx

```diff
@@ -0,0 +1,64 @@
+import Chat from "components/chat";
```

Comment:
this file needs a license header

Reply:
Added it https://github.com/mentorpal/mentor-client/pull/89/commits/7403c9a9c4ee03ee11ad62900b169de15cdc0408